### PR TITLE
Task 869: Get Telemetry

### PR DESCRIPTION
**Pull request by sebastian-j-ibanez**
### Changes Made
Welcome to my work on ticket ID869. Let me know if there is anything that should be changed. :mate: 
#### main.go
:green_circle: Added /requestTelemetry route. Sends request to Space CNDH via Ground Uplink/Downlink's /send/ route.
#### scripts.js
:green_circle: Added getTelemetryBtn event listener which makes a GET request to /requestTelemetry. Send alert when it is successful.
:green_circle: Changed all double quotations to single quotations for consistency.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -246,6 +246,20 @@ func serveCSS(c *gin.Context) {
...
 
+func requestTelemetry(c *gin.Context) {
+	c.Header("content-type", "application/json")
+	uri := fmt.Sprintf("http://%s:8080/send/", listIPs[4])
+
+	// Create JSON
+	json := "{\"verb\":\"GET\",\"uri\":\"http://" + listIPs[2] + ":8080/send/\"}"
+	body := strings.NewReader(json)
+
+	res, err := http.NewRequest("POST", uri, body)
```
Comment (Inventhrice) [inline main.go:257]: I am unsure as to what this does. I don't see:
- why you have written c.Header(application/json) when that is what is going to be in the response
- Where the return code for the function is

**Review by Inventhrice — CHANGES_REQUESTED** [review]
Please refer to the comments! You are half way there though!

File: main.go (hunk 1 above)
Comment (sebastian-j-ibanez) [reply main.go:257]: 1. That was unfortunately left over from another iteration of the function.
2. I goofed, and did not set the response status code.

Comment (sebastian-j-ibanez) [reply main.go:256]: I should be solved in the latest commit

**Review by Inventhrice — APPROVED** [review]
It looks great!

**Pull request merged**